### DISCUSSION TITLE
Different cost for each type of cell

### DISCUSSION THREAD

**Discussion by PhoneixS** [MaxRobinsonTheGreat/EvolutionSimulatorV2#16] [Ideas and Suggestions]
I think it could be nice if the cells have different costs of "live". For example, a moving cell costs twice as a mouth in reproduction.

I think this would better mimic nature (or make "other" natural orders possible), in nature moving is costly (that's why you usually don't have moving plants in my opinion). Also, predator cells (I imagine them as having poison) have a big cost in nature (that's why many venous animals prefer to display it instead of using it if they can) so maybe they can cost 3 times the mouth cost.

And if you could change the cost of each cell in the simulation options, it could open a new range of possibilities for different environments. For instance, this could make the camo cell (#5) a viable cell strategy as it could cost less to have a camo cell than a predator cell but have the same effect of deterring predators.

As an alternative, instead of a different cost in reproduction, it could make aging faster but I think this would have the same effects and is probably harder to implement.

**Comment by doo158** [comment]
fantastic. developer if you read both my post and this guy's, definately implement this over my cell probability idea

**Comment by theomgdev** [comment]
I did it :) #107
